core/templates/services/assets-backend-api.service.ts

```diff
@@ -243,6 +285,12 @@ angular.module('oppia').factory('AssetsBackendApiService', [
           _saveAudio(explorationId, filename, rawAssetData, resolve, reject);
         });
       },
+      saveMathImage: function(resampledFile, filename, entityType, entityId) {
```

Comment:
maybe saveMathExpresionImage? (Can we generalize this to saveSvgImage? not sure*)

Reply:
done